editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java

```diff
@@ -174,6 +173,28 @@ private boolean isWARProject(IProject project, IProgressMonitor monitor) {
 		return CamelProjectConfigurator.WAR_PACKAGE.equalsIgnoreCase(m2prj.getPackaging());
 	}
 
+	private boolean isInvalidCamelFacetVersion(ProjectConfigurationRequest request) throws CoreException {
+		IProject project = request.getProject();
+		IFacetedProject fproj = ProjectFacetsManager.create(project);
+		if (fproj != null) {
+			return !fproj.getInstalledVersion(camelFacet).getVersionString().equalsIgnoreCase(DEFAULT_CAMEL_FACET_VERSION);
```

Comment:
if using a 3.0 version I think that it would be easier in the future in case we want to provide another version of the facet. Becaue  we can write already is version < do the migration otherwise we will have to think to update this line.